http/routes.go

```diff
@@ -9,16 +9,20 @@ func (a *AppServer) authenticatedRoutes() *httprouter.Router {
 
 	// User
 	router.GET("/api/v1/users/:id", a.GetUserHandler())
+	router.GET("/api/v1/users/:id/courses", a.GetMyCoursesHandler())
 	router.GET("/api/v1/me", a.GetMeHandler())
 
 	// Courses
 	router.GET("/api/v1/courses/:id", a.GetCourseHandler())
+	router.GET("/api/v1/courses/:id/users", a.GetMembersHandler())
+	router.POST("/api/v1/courses/:id/users/subscribe", a.AddMembersHandler())
+	router.POST("/api/v1/courses/:id/users/unsubscribe", a.RemoveMembersHandler())
 	router.GET("/api/v1/courses", a.GetAllCoursesHandler())
 
 	// CourseEntries
-	router.POST("/api/v1/courses/:courseID/entries", a.PostCourseEntryHandler())
-	router.PUT("/api/v1/courses/:courseID/entries/:entryID", a.PutCourseEntryHandler())
-	router.DELETE("/api/v1/courses/:courseID/entries/:entryID", a.DeleteCourseEntryHandler())
```

Comment:
I think leaving `courseID` here is better to know what ID one is referring to in the respective handlers.